frontend/javascripts/oxalis/model/sagas/isosurface_saga.js

```diff
@@ -171,8 +197,35 @@ function* maybeLoadIsosurface(
   }
 }
 
+function* downloadActiveIsosurfaceCell(): Saga<void> {
+  const sceneController = getSceneController();
+  const geometry = sceneController.getIsosurfaceGeometry(currentIsosurfaceCellId);
+
+  const stl = exportToStl(geometry);
+
+  // Encode isosurface and cell id property
+  binaryIsosurfaceMarker.forEach((marker, index) => {
+    stl.setUint8(index, marker);
+  });
+  stl.setUint32(3, currentIsosurfaceCellId, true);
```

Comment:
The `3` should be extracted from this and the next method, maybe in a similar way as the binaryIsosurfaceMarker :)